site/en/docs/extensions/support-feedback/submit-a-feature-request/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Submit a feature request
+description: 'TBD'
+date: 2023-10-11
+tags:
+  - extensions
+---
+
+## Submit a feature request
+
+If you identify a feature (especially if it's related to an experimental API) that could improve the extensions, file a request as described below.
+
+1.  Before submitting a feature request, [search the Chromium issue tracker](/docs/extensions/support-feedback/find-a-bug) to verify that your feature has not already been proposed.
```

Comment:
I'm wondering if we should include something about the [WECG](https://github.com/w3c/webextensions/issues) issues?

@oliverdunk @sebastianbenz any thoughts? I don't think this is a blocker, it just occurred to me since they discuss many proposals and feature requests

Reply:
I asked Oliver that in the Docs draft. This is what he said we should do.